api.py

```diff
@@ -27,8 +27,27 @@ def fetch_posts_and_boosts(
     seen_post_urls = set()
     total_posts_seen = 0
 
-    # Iterate over our home timeline until we run out of posts or we hit the limit
-    response = mastodon_client.timeline(min_id=start)
+    # If timeline name is specified as hashtag:tagName or list:list-name, look-up with those names,
+    # else accept 'federated' and 'local' to process from the server public and local timelines.
+    #
+    # We default to 'home' if the name is unrecognized
+    if ":" in timeline:
+        timelineType, timelineName = timeline.split(":", 1)
+    else:
+        timelineType = timeline
+
+    if timelineType == "hashtag":
+        response = mastodon_client.timeline_hashtag(timelineName, min_id=start)
+    elif timelineType == "list":
+        response = mastodon_client.timeline_list(timelineName, min_id=start)
+    elif timelineType == "federated":
+        response = mastodon_client.timeline_public(min_id=start)
+    elif timelineType == "local":
+        response = mastodon_client.timeline_local(min_id=start)
+    else:
+        response = mastodon_client.timeline(min_id=start)
```

Comment:
This seems like a reasonable default, but it gets masked when the digest is rendered. If I type in `-f foo` the details read:

> from timeline “foo"

Would it make sense to do some of this validation in `run.py` so that the timeline name can fall to a default before being passed to `render_digest()`?

Reply:
Final commit just went in that does some loose validation at input time, so that `home` is explicitly reset for unknown inputs.

Thank for the feedback!